### PR TITLE
RDoc-1467 - Document Memory.LowMemoryCommitLimitInMb and Memory.MaxFreeCommittedMemoryToKeepInMb

### PR DESCRIPTION
**Pull request by grisha-kotler**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/server/configuration/memory-configuration.markdown [hunk 1]
```diff
@@ -10,6 +10,16 @@ The minimum amount of available memory RavenDB will attempt to achieve (free mem
 
 {PANEL/}
 
+{PANEL:Memory.LowMemoryCommitLimitInMb}
+
+The minimum amount of available commited memory RavenDB will attempt to achieve (free commited memory lower than this value will trigger low memory behavior). Value is in MB.
+
+- **Type**: `int`
+- **Default**: `512MB`
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/configuration/memory-configuration.markdown]: Let's get rid of the `MB`

File: Documentation/4.0/Raven.Documentation.Pages/server/configuration/memory-configuration.markdown [hunk 2]
```diff
@@ -19,3 +29,13 @@ EXPERT: The minimum amount of committed memory that RavenDB will attempt to ensu
 - **Scope**: Server-wide only
 
 {PANEL/}
+
+{PANEL:Memory.MaxFreeCommittedMemoryToKeepInMb}
+
+EXPERT: The maximum amount of committed memory that RavenDB will attempt to ensure remains available. Reducing this value too much may cause RavenDB to fail if there is not enough memory available for the operation system to handle operations.
+
+- **Type**: `int`
+- **Default**: `128MB`
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/configuration/memory-configuration.markdown]: Let's remove the `MB` and write in description that the value is in `MB`

Comment (grisha-kotler) [reply Documentation/4.0/Raven.Documentation.Pages/server/configuration/memory-configuration.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/server/configuration/memory-configuration.markdown (hunk 1 above)
Comment (grisha-kotler) [reply Documentation/4.0/Raven.Documentation.Pages/server/configuration/memory-configuration.markdown]: done

**Pull request merged**